src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -25,6 +25,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Helpers.cs" />
+    <Compile Include="Microsoft\Win32\SafeHandles\SafeWaitHandle.cs" />
```

Comment:
I see, for this case, what we have been doing is to not move the tests even though the APIs move in contracts. By doing this, everything *should* still work because System.Runtime.Handles tests will have a reference to its own pkgproj, and this will get the asset for netcoreapp1.0 which will still contain the APIs in it.

---
In reply to: [76884498](https://github.com/dotnet/corefx/pull/11229#discussion_r76884498) [](ancestors = 76884498)